src/sentry/seer/anomaly_detection/store_data.py

```diff
@@ -50,7 +49,7 @@ def format_historical_data(data: SnubaTSResult) -> list[TimeSeriesPoint]:
     return formatted_data
 
 
-def send_historical_data_to_seer(alert_rule: AlertRule, user: User) -> BaseHTTPResponse:
```

Comment:
I moved the feature check to `logic.py` which is the only thing I was using `user` for